### PR TITLE
OADP-2133: adding note for using VolumeOptionsForStorageClass with Ce…

### PR DESCRIPTION
**Pull request by anarnold97**
…ph only. Editing the DPA CR and removing related numbered bulletpoint

[OADP-2132](https://issues.redhat.com/browse/OADP-2133): Adding IMPORTANT admonition for using VolumeOptionsForStorageClass with Ceph only. Editing the DPA CR and removing related numbered bulletpoint for OADP 1.2.0.

Version(s):

Enterprise 4.10 → branch/enterprise-4.10
Enterprise 4.11 → branch/enterprise-4.11
Enterprise 4.12 → branch/enterprise-4.12
Enterprise 4.13 → branch/enterprise-4.13
Enterprise 4.14 → branch/enterprise-4.14
Issue:

https://issues.redhat.com/browse/OADP-2133

Version(s):

Issue:

Link to docs preview:

[oadp-using-data-mover-for-csi-snapshots_backing-up-applications](https://61630--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
@akarol && @weshayutin - please review and approve if you are ok with the changes

File: modules/oadp-using-data-mover-for-csi-snapshots.adoc [hunk 1]
```diff
@@ -16,6 +16,10 @@ Data Mover supports backup and restore of CSI volume snapshots only.
 
 In OADP 1.2 Data Mover `VolumeSnapshotBackups` (VSBs) and `VolumeSnapshotRestores` (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously `InProgress`. After all async plugin operations are complete, the backup is marked as complete.
 
+[IMPORTANT]
+====
+The `VolumeOptionsForStorageClass` label is for use with CephFS only.
```
Comment (weshayutin) [inline modules/oadp-using-data-mover-for-csi-snapshots.adoc:21]: oh sorry sir.. that is not a label.. it's a parameter of the dpa.

[whayutin@thinkdoe ~]$ oc explain dataprotectionapplications.oadp.openshift.io.spec.features.dataMover
KIND:     DataProtectionApplication
VERSION:  oadp.openshift.io/v1alpha1

RESOURCE: dataMover <Object>

DESCRIPTION:
     Contains data mover specific configurations

FIELDS:
   credentialName	<string>
     User supplied Restic Secret name

   enable	<boolean>
     enable flag is used to specify whether you want to deploy the volume
     snapshot mover controller

   maxConcurrentBackupVolumes	<string>
     the number of batched volumeSnapshotBackups that can be inProgress at once,
     default value is 10

   maxConcurrentRestoreVolumes	<string>
     the number of batched volumeSnapshotRestores that can be inProgress at
     once, default value is 10

   pruneInterval	<string>
     defines how often (in days) to prune the datamover snapshots from the
     repository

   snapshotRetainPolicy	<Object>
     defines the parameters that can be specified for retention of datamover
     snapshots

   timeout	<string>
     User supplied timeout to be used for VolumeSnapshotBackup and
     VolumeSnapshotRestore to complete, default value is 10m

   volumeOptionsForStorageClasses	<map[string]Object>
     defines configurations for data mover volume options for a storageClass

Comment (anarnold97) [reply modules/oadp-using-data-mover-for-csi-snapshots.adoc]: changed sorry

**Comment by weshayutin** [comment]
@shubham-pampattiwar @sseago does this doc update send the correct message re: volumeOptions in the DPA?

**Comment by anarnold97** [comment]
@sseago - please can you clarify what you meant by this comment: "_So I think it could be used for non-CephFS cases if there are specific concerns or cluster configuration needs, but the doc example in the PR is a CephFS-specific example, and most users won't need it in other cases._ "

**Comment by sseago** [comment]
@anarnold97 The example given for this field in the document is a CephFS-specified use case, but the field itself could be used for any storage type, although I think CephFS is really the only case we know of where most users wilil need it.

**Pull request closed without merging**